flowmachine/flowmachine/core/server/query_schemas/aggregate_network_objects.py

```diff
@@ -0,0 +1,47 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from marshmallow import Schema, fields, post_load
+from marshmallow.validate import OneOf, Length
+
+from flowmachine.features import AggregateNetworkObjects
+from .base_exposed_query import BaseExposedQuery
+from .custom_fields import AggregationUnit
+
+__all__ = ["AggregateNetworkObjectsSchema", "AggregateNetworkObjectsExposed"]
+
+
+class AggregateNetworkObjectsSchema(Schema):
+
+    start_date = fields.Date(required=True)
```

Comment:
Needs to have the right fields, one of which is a bit tricky because it should refer to another schema. See FlowsSchema, or ModalLocationSchema for an example of how to do that.